src/Swagger/LoginDecorator.php

```diff
@@ -41,7 +41,7 @@ public function normalize($object, string $format = null, array $context = []):
 
         $docs['paths']['/api/users']['post']['responses']['200']['content']['application/json']['schema']['$ref'] = '#/components/schemas/User-read';
         $docs['paths']['/api/users']['post']['responses']['200']['content']['text/html']['schema']['$ref']        = '#/components/schemas/User-read';
-        unset($docs['components']['schemas']['User:83fb6eab7febe7ac9423776db8677557']);
+        //unset($docs['components']['schemas']['User:83fb6eab7febe7ac9423776db8677557']);
```

Comment:
I believe this was only meant to be commented out for local testing, but I need a second opinion.